rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -546,15 +569,32 @@ pub fn tags_to_lanes(
 ) -> Result<RoadFromTags, RoadError> {
     let mut warnings = RoadWarnings::default();
 
+    // Early return if we find unimplemented tags.
     unsupported(tags, locale, &mut warnings)?;
 
-    let mut road: RoadBuilder = RoadBuilder::from(tags, locale, &mut warnings)?;
+    // Parse lane count schemas first.
+    let oneway =
+        Oneway::from(tags.is_any("oneway", &["yes", "-1"]) || tags.is("junction", "roundabout"));
+    let busway = BuswayScheme::new(tags, locale, &oneway, &mut warnings);
+    // let bus_lanes = BusLanes::from(tags, locale, &oneway, &mut warnings);
+    // TEMP: lets use the lanes:bus schema to summarise bus lanes for the lanes scheme.
+    let lanes_bus = LanesBusScheme {
+        forward: busway.forward_direction.map_or(0, |o| if o.is_some() { 1 } else { 0 }),
+        backward: busway.backward_direction.map_or(0, |o| if o.is_some() { 1 } else { 0 }),
```

Comment:
this construct seems fishy, probably due to comments above?

Reply:
Mm, it is a little. I am using `lanes_bus.backward` to count lanes on the backward side of the road (which `BuswayScheme` will switch direction if it needs to). I suppose it depends (if we even need to support such a thing) how `lanes:bus` would represent the contraflow bus lane on the unexpected side of the road situation.